mypy/test/teststubgen.py

```diff
@@ -64,6 +64,23 @@ def test_packages_found(self) -> None:
             finally:
                 os.chdir(current)
 
+    def test_module_not_found(self) -> None:
+        current = os.getcwd()
+        with tempfile.TemporaryDirectory() as tmp:
+            try:
+                import io
+                capturedOutput = io.StringIO()
+                sys.stdout = capturedOutput
```

Comment:
I would move the parts that should not fail (the two above lines) above the `try`.